Purchases/Purchasing/RCStoreKitRequestFetcher.h

```diff
@@ -15,20 +15,13 @@ typedef void(^RCFetchProductsCompletionHandler)(NSArray<SKProduct *> *products);
 
 typedef void(^RCFetchReceiptCompletionHandler)(void);
 
-@class SKProduct, SKProductsRequest;
-
-@interface RCProductsRequestFactory : NSObject
-- (SKProductsRequest *)requestForProductIdentifiers:(NSSet<NSString *> *)identifiers;
+@interface RCReceiptRefreshRequestFactory : NSObject
 - (SKReceiptRefreshRequest *)receiptRefreshRequest;
 @end
 
-@interface RCStoreKitRequestFetcher : NSObject <SKProductsRequestDelegate>
-
-- (nullable instancetype)initWithRequestFactory:(RCProductsRequestFactory *)requestFactory;
-
-- (void)fetchProducts:(NSSet<NSString *> *)identifiers
-           completion:(RCFetchProductsCompletionHandler)completion;
+@interface RCStoreKitRequestFetcher : NSObject
 
+- (nullable instancetype)initWithRequestFactory:(RCReceiptRefreshRequestFactory *)requestFactory;
 - (void)fetchReceiptData:(RCFetchReceiptCompletionHandler)completion;
```

Comment:
in part 2, we'll finish killing this file. Part 1 only removes stuff related to fetching SKProducts.

Reply:
Noice 💀